docs/how-to-guides/software-management/public-release-process.md

```diff
@@ -144,7 +147,8 @@ Anaconda leverages caching in many places which can cause issues. If you are get
 
 * Ensure the OS on the machine you are building the release on is the appropriate operating system (Mac OS 11.6 or Ubuntu 18 LTS). 
 
-    * If you do not have access to a Ubuntu 18 LTS, you can ssh into prog28. 
+    ???+ Info "For internal devs"
+        If you do not have access to an Ubuntu 18 LTS, follow the steps [here](https://code.chs.usgs.gov/asc/ASC_Software_Docs/-/blob/main/Software%20Management/Public%20Release%20Process%20-%20Internal%20Additions.md?ref_type=heads#step-4-create-the-builds-for-anaconda-cloud) to access an EC2 instance running Ubuntu.
```

Comment:
just for consistency, I kind of see @jlaura's point about not including things that are just for internal devs in these. Maybe there exists an elegant way to handle these cases since I think I still prefer these being useful to both external and internal readers.

Reply:
Hm if that's the case then should Steps 7 - 9 be moved to the internal doc? Should we still have a link to the internal doc or are we relying on pointing internal devs to the internal doc first, which would then lead them to the public facing doc?

Reply:
I think rn the latter is the case. I think we just need to point to people internally to internal stuff for whatever pertains to that. Having links to internal stuff, outside the aesthetics of it as a reader, is probably asking for trouble down the line.